creator-node/src/services/prometheusMonitoring/prometheus.constants.js

```diff
@@ -0,0 +1,54 @@
+const promClient = require('prom-client')
+
+const METRIC_PREFIX = 'cn_'
+
+/**
+ * @notice Counter and Summary metric types are currently disabled, since we should almost never be using them.
+ *  - Gauge should always be used over Counter since it can be decreased, with no performance overhead.
+ *  - Histogram should always be used over Summary, per docs below
+ * https://prometheus.io/docs/tutorials/understanding_metric_types/
+ */
+const MetricTypes = Object.freeze({
+  GAUGE: promClient.Gauge,
+  HISTOGRAM: promClient.Histogram
+  // COUNTER: promClient.Counter,
+  // SUMMARY: promClient.Summary
+})
+
+const MetricNames = Object.freeze({
+  STORAGE_PATH_SIZE_GAUGE: 'storage_path_size_bytes',
+  STORAGE_PATH_USED_GAUGE: 'storage_path_used_bytes',
```

Comment:
I like how you didn't use `_current` here since it wasn't needed.

But that definitely opens up a future conversation of when `_current` is ideal. Perhaps only for "in-progress" stats? Either way, we can have that discussion around a future gauge.

Reply:
right, given that all gauges measure "current" state, didn't think it was required, except for the specific cases where it is needed for extra clarification

Reply:
Nice, I like this approach! I'm on board! :D